protocol/protocol.go

```diff
@@ -24,19 +24,26 @@ import (
 func (m *Manager) routerInitializer(netConnection *netadapter.NetConnection) *routerpkg.Router {
 	router := routerpkg.NewRouter()
 	spawn("newRouterInitializer-startFlows", func() {
-		if m.context.ConnectionManager().IsBanned(netConnection) {
+		isBanned, err := m.context.ConnectionManager().IsBanned(netConnection)
+		if err != nil {
+			panic(err)
```

Comment:
this shouldn't panic since it's ok for you to not know an address of an inbound connection